src/components/AuthProvider/AuthProvider.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { computeAuthState } from "components/AuthProvider/AuthProvider";
+import { AUTHENTICATED } from "config/localStorageKeys";
+
+afterEach(() => localStorage.removeItem(AUTHENTICATED));
+
+describe("computeAuthState", () => {
+  it("should return the correct authentication state", () => {
+    expect(computeAuthState(false)).toBe(false);
+    expect(computeAuthState(true)).toBe(true);
+
+    localStorage.setItem(AUTHENTICATED, "true");
```

Comment:
Storing a boolean `authenticated` in the localStorage doesn't make a lot of sense.
A JWT token is what you would want to store, the _presence_ of said token indicates that you are logged in (expiry is another thing)

Reply:
@patrixr at the moment the intention is to create a barebones skeleton implementation. So there are no API calls including /verify. In future PR will introduce authToken.

Reply:
gotcha, in that case you can replace the "authenticated" key by "jwt" (or something alike) and put in a dummy value ("true" still works). and have `computeAuthState` look at the presence of it. Just to avoid this "authenticated" key which doesn't make any sense